src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form"
+    >
+      <fieldset class="form__fieldset">
+        <legend class="form__fieldset-legend">Personal information</legend>
+        <label class="form__lable">
```

Comment:
fix this typo all over the page
```suggestion
        <label class="form__label">
```